libs/components/lookup/src/lib/modules/selection-modal/selection-modal.component.scss

```diff
@@ -5,6 +5,16 @@
   padding: $sky-space-xl 0;
 }
 
+::ng-deep .hover-rows {
```

Comment:
To avoid the `:ng-deep`, is there an opportunity to add this feature/fix to repeater, directly?

Reply:
Selection modal adds a click handler for single select mode; it's not a baseline feature of repeater.